src/index.js

```diff
@@ -0,0 +1,9 @@
+import Menu from './Menu.js';
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
FWIW, instead of the content loaded listener on the public components, I just moved the script to the bottom of the page. lmk if you don't like that when I demo it.